sources/System/Timer/TimerSystem.cpp

```diff
@@ -4,8 +4,10 @@
 
 #include "TimerSystem.hpp"
 #include "Component/Timer/TimerComponent.hpp"
+#include "Drawables/2D/Text.hpp"
 
 using namespace std::chrono_literals;
+namespace RAY2D = RAY::Drawables::Drawables2D;
```

Comment:
why ? you are not using any drawables

Reply:
I used them at a moment, not anymore, deleted now